common/logger/src/struct_log.rs

```diff
@@ -281,13 +284,16 @@ pub fn struct_logger_set() -> bool {
 }
 
 /// Initializes struct logger from STRUCT_LOG_FILE env var.
-/// If STRUCT_LOG_FILE is set, STRUCT_LOG_UDP_ADDR will be ignored.
+/// If STRUCT_LOG_FILE is set, STRUCT_LOG_TCP_ADDR will be ignored.
 /// Can only be called once
 pub fn init_struct_log_from_env() -> Result<(), InitLoggerError> {
     if let Ok(file) = env::var("STRUCT_LOG_FILE") {
         init_file_struct_log(file)
-    } else if let Ok(udp_address) = env::var("STRUCT_LOG_UDP_ADDR") {
-        init_udp_struct_log(udp_address)
+    } else if let Ok(address) = env::var("STRUCT_LOG_TCP_ADDR") {
+        init_tcp_struct_log(address)
+    } else if let Ok(address) = env::var("STRUCT_LOG_UDP_ADDR") {
+        // Remove once all usages of STRUCT_LOG_UDP_ADDR are transferred over
```

Comment:
This whole "logging", with no logger is tricky.  So, I added metrics to take its place since the revision you looked at.  I did a grep of all of our files for TCP_ADDR vs UDP_ADDR, and I think we should be fine, between the two PRs